src/FixedDataTableTranslateDOMPosition.js

```diff
@@ -20,9 +22,18 @@ function FixedDataTableTranslateDOMPosition(/*object*/ style, /*number*/ x, /*nu
     style.left = x + 'px';
     style.top = y + 'px';
   } else {
+    if (BrowserSupportCore.hasCSSTransforms() && Locale.isRTL()) {
+      x *= Locale.DIR_SIGN();
+    }
+
     translateDOMPositionXY(style, x, y);
   }
 
+  if (Locale.isRTL()) {
+    style.right = style.left;
+    style.left = 'auto';
```

Comment:
Is this needed? I thought flipping the x value from the condition above will mean that `translateDOMPositionXY` will also flip the right and left values.

Reply:
Depending on browser support, either `translate3d` is used or the `style.left` is changed. In either case, it would be safe to flip the `style.left` and `style.right` values